test/simplelabel/simplelabel.str

```diff
@@ -0,0 +1,202 @@
+#include "traindir/auxiliary/segway.inc"
+
+GRAPHICAL_MODEL model_seg
```

Comment:
If possible, I would remove the unnecessary pre-processor conditional macros from this since this is going to be a simpletest. It would be nice to have as a quick reference in terms of whats necessary to get string labels to work.

Reply:
The structure file uses around a dozen pre-processor macros, so I don't think substituting this for individual define statments would make the test any simpler. While these macros could be substituted directly in the file, I don't think that's needed when all other tests are using this include statement in the structure file.